storage/tests/unit/test__helpers.py

```diff
@@ -62,6 +73,25 @@ def test_reload(self):
         # Make sure changes get reset by reload.
         self.assertEqual(derived._changes, set())
 
+    def test_reload_w_user_project(self):
+        user_project = 'user-project-123'
+        connection = _Connection({'foo': 'Foo'})
+        client = _Client(connection)
+        derived = self._derivedClass('/path', user_project)()
+        # Make sure changes is not a set, so we can observe a change.
```

Comment:
The comment is "not a set", i.e., not an instance of `set`.  It gets replaced during `reload` with an empty `set` instance.